content/support/_index.md

```diff
@@ -0,0 +1,78 @@
+  
+  ## Learn more about who supports our work
+
+  [Institutional funders and members since launch →](/support/funders-and-members#major-donors-and-funders)
+
+  [Leaders in Action →](/support/leaders-in-action/#our-current-leaders)
+
+  [State College regional bureau supporters →](/support/state-college/#our-major-donors)
+"""
+
+[[blocks]]
+title = "Transparency"
+layout = "text-2"
+copy="""
+  {{<featured/large>}}Spotlight PA is a nonpartisan, independent newsroom paid for by grants from foundations and donations from individuals committed to robust investigative reporting. Spotlight PA editors and reporters operate independently of our funders and maintain editorial control over all of the content they produce. Funders do not have input into the selection of which stories Spotlight PA pursues or the reporting process for those stories. Funders do not review any content before publication or have any special access to reporters or newsroom leadership. In short, we answer to you, the public.{{</featured/large>}}
+
+  {{<featured/large>}}Funds for Spotlight PA are donated to the nonprofit <a href="https://www.lenfestinstitute.org/" class="text-yellow">Lenfest Institute for Journalism</a>, which then directs them to our work. As part of our gift acceptance policy, and consistent with our promise to be fully transparent, we do not accept anonymous donations.{{</featured/large>}}
+"""
++++
+{{<featured/large>}}Spotlight PA produces investigative and public-service journalism that gets results, and shares that work at no cost with communities across Pennsylvania. But we can't do it without your support. Here are the several ways you can sustain our newsroom with a tax-deductible donation today:{{</featured/large>}} 
+
+{{<about/action-link href="https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&campaign=7015G000001Z88SQAS">}}Donate With Credit Card  →{{</about/action-link>}}
+
+{{<about/action-link href="https://www.paypal.com/donate?hosted_button_id=BWKY4YD8CU9ML">}}Donate Via PayPal  →{{</about/action-link>}}
+
+## Mail a Check
+
+Make a check out to Spotlight PA and send it to:
+
+**Spotlight PA** <br>
+**228 Walnut St. #11728** <br> 
+**Harrisburg, PA 17108-1728** <br>
+
+## Use Your Donor-Advised Fund
+
+Recommend a gift via your donor-advised fund. Please direct it to The Lenfest Institute for Journalism Special Asset Fund of The Philadelphia Foundation, tax ID number 04-3731829, and be sure to mention that the gift is for Spotlight PA in the paperwork.
+
+## Donate Shares of Stock
+
+For gifts of stocks and securities, contact [joanna@spotlightpa.org](joanna@spotlightpa.org). 
```

Comment:
This needs mailto:

Reply:
resolved with https://github.com/spotlightpa/poor-richard/pull/338/commits/32a379b2b2719d4908f5430ce2ca49d44a837ea3